tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -80,90 +80,111 @@ object Discover {
   }
 
   /** Tests whether the clang compiler is recent enough.
-   *  It is determined through looking up a built-in #define which
-   *  is more reliable than testing for a specific version.
+   *  Clang must be greater or equal to the minumum version and
+   *  by checking for a built-in #define which is more reliable
+   *  than testing for a specific version.
    */
   private[scalanative] def checkThatClangIsRecentEnough(
       pathToClangBinary: Path): Unit = {
-    def maybePath(p: Path) = p match {
-      case path if Files.exists(path) => Some(path.abs)
-      case none                       => None
-    }
 
-    def definesBuiltIn(
-        pathToClangBinary: Option[String]): Option[Seq[String]] = {
+    // this could be obsolete with our current minimum version
+    def checkDefinesBuiltIn(clang: String): Unit = {
+      val defineStrs = "__DECIMAL_DIG__ __LDBL_DECIMAL_DIG__"
```

Comment:
Inded it might be obsolete check.`__DECIMAL_DIG__` preprocessor definition exists since clang 3.7
https://github.com/llvm/llvm-project/commit/4992099b42e9bb6faceb61ed093155bf2ebe7157

Probably we can remove this

Reply:
Ok, I just removed that in the first commit.

Reply:
Thanks for checking. I thought we didn't need it but I also didn't know anything about it.